Neos.Media/Classes/Domain/Model/AssetSource/Neos/NeosAssetProxyQueryResult.php

```diff
@@ -100,6 +101,7 @@ public function current(): ?AssetProxyInterface
     /**
      * @return void
      */
+    #[ReturnTypeWillChange]
```

Comment:
In other classes you used the FQN. But it's up to you!

```suggestion
    #[\ReturnTypeWillChange]
```